static/js/ticketView/activityContainer.js

```diff
@@ -1,6 +1,106 @@
-const TicketActivityContainer = {
-    methods: {
+function convertDatetime(datetimeStr) {
```

Comment:
this looks like an overkill func :)
if really needed here - ok. Else try make your api return date as `d.isoformat()` and in js make `new Date(d + 'Z').toLocalString()` - it will produce a formatted date string based on user's browser locale and `+ 'Z'` meant to make the iso-date a timezone-aware date